tests/DependencyInjection/RabbitMqConsumerHandlerExtensionTest.php

```diff
@@ -18,140 +21,128 @@ protected function getContainerExtensions(): array
 	}
 
 	/**
-	 * @return mixed[][]
+	 * @return mixed[][]|\Generator
 	 */
-	public function defaultConfigurationValuesProvider(): array
+	public function configureContainerParameterDataProvider(): Generator
 	{
-		return [
-			[
-				RabbitMqConsumerHandlerExtension::CONTAINER_PARAMETER_STOP_CONSUMER_SLEEP_SECONDS,
-				1,
-			],
-			[
-				RabbitMqConsumerHandlerExtension::CONTAINER_PARAMETER_ENTITY_MANAGER_CLEAR,
-				true,
-			],
+		yield 'default stop_consumer_sleep_seconds value' => [
+			'configurationValues' => [],
```

Comment:
I think we should use just `configuration` for these, see https://github.com/VasekPurchart/Tracy-Blue-Screen-Bundle/pull/30#discussion_r1162978415 .

Reply:
Renamed to `configuration`.